.github/workflows/weekly-ui-tests.yml

```
@@ -0,0 +1,78 @@
+name: Weekly UI Test - Enquiry Form
+
+on:
+  #  workflow_dispatch:
+
+  schedule:
+    # Monday at 7 AM UTC - https://cron.help/#0_12_*_*_MON
+    - cron: "0 7 * * MON"
+  #pull_request:
+  #  branches:
+  #    - main
+  workflow_dispatch:
+    inputs:
+      TARGET_URL:
+        type: string
+        description: "Target URL"
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+defaults:
+  run:
+    shell: pwsh
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v2
+        with:
+          path: ./.github
+
+      - name: Azure CLI - Login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: keyVault - Get Secret Key note
+        id: KeyVaultSecrets
+        run: |
+          # KV - Secret
+          $SECRET_KEY_TO_BYPASS_RECAPTCHA = (az keyvault secret show --name SECRET-KEY-TO-BYPASS-RECAPTCHA --vault-name ${{ env.KEY_VAULT }} --query value -o tsv)
+          echo "::add-mask::$SECRET_KEY_TO_BYPASS_RECAPTCHA"
+          echo "SECRET_KEY_TO_BYPASS_RECAPTCHA=$SECRET_KEY_TO_BYPASS_RECAPTCHA" >> $env:GITHUB_OUTPUT
+          Write-Host '✅ KV - Secret retrieved'
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Install Playwright Browsers
+        run: yarn playwright install --with-deps
+
+      - name: Run Playwright tests
+        run: yarn playwright test
+        env:
+          TARGET_URL: ${{ inputs.TARGET_URL || env.TARGET_URL }}
+          SECRET_KEY_TO_BYPASS_RECAPTCHA: ${{ steps.KeyVaultSecrets.outputs.SECRET_KEY_TO_BYPASS_RECAPTCHA }}
+
+      - uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 1
```

```suggestion
          retention-days: 14
```
2 weeks of reports should be enough

---

done